### PR TITLE
Want to merge feature branch to master branch

### PR DESCRIPTION
**Pull request by Kingobaino1**
**In this project**
- added navbar
- added header
- added body 
- added contents of body
- added a side-bar
- added footer

### CODE REVIEW

File: CSS/style.css [hunk 1]
```diff
@@ -0,0 +1,606 @@
...
+.grid_icons img {
+  width: 20px;
+  height: 20px;
+}
+
+.right_margin {
+  margin-right: 10px;
+}
+
+#marg_bot {
```
Comment (kevcap) [inline CSS/style.css:514]: - [ ] [OPTIONAL] Use class instead 🔧

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️


#### Overall the project looks really nice and I can see that you guys are working hard on it, and don't worry, you're almost done, there are just some required changes that you guys have to implement to proceed:


_Note_: That changes marked as [OPTIONAL] are not required to approve your project, but I strongly recommend that you guys go for that extra mile, in order to get more knowledge and have a better project to add at your portfolio.

## [REQUIRED CHANGES]
#### Good practices REQUIREMENTS (Those changes to make you code like a pro 👽)⤵
- [x] Please remove that images folder who is inside the CSS folder.
- [x] Kindly remember to remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
Take a look at this example:
image
Your HTML will be something like this: 
image
Apply the same logic above in **ALL** those repeated rules listed below:
image

#### Clone REQUIREMENTS 👥 (Those adjusts to make your page really look like the real one)⤵
- [x] Your page has some overlapping, please take care of it 💪
image
- [x] It uses [`<aside>` tag](https://www.w3schools.com/tags/tag_aside.asp) for `Editor's picks` section on the righmost side. , kindly remove that `<div>`. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/aside_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/))
image



## [OPTIONAL CHANGES]
- [x] Add the title of the original page;
- [ ] Add a favicon to make it more similar to the original page;

#### Please make the required changes and resubmit for a new code review once you are ready💪

## As always, I wish you all the very best and see you both at the top🚀

👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question about this review? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

## Hi team, good job on making the previously required changes!
Your page meets most of the requirements  👏 
You just have to implement some minor changes in order to move on
 
## [REQUIRED CHANGES]
#### Clone REQUIREMENTS 👥 (Those adjusts to make your page really look like the real one)⤵
- [x] Your page still has some overlapping, please take care of it 👀
image
- The original one: 
image
Tip: Check paddings, margings, position and other spacing options to solve that issue 👷‍♂️

## [OPTIONAL CHANGES]
- [x] Add a favicon to make it more similar to the original page;
- [ ] Fix the font sizing to make it more similar to the original page;

## Please make the required changes and resubmit for a new code review once you are ready💪

## As always, I wish you all the very best and see you both at the top🚀


👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

  
<sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

## Hi team, good job on making the previously required changes, however looks like you guys fixed the overlapping in the wrong way (using font-size), please take care of it! 💪
You just have to implement some minor changes in order to move on
 
## [REQUIRED CHANGES]
#### Clone REQUIREMENTS 👥 (Those adjusts to make your page really look like the real one)⤵
- [x] Now your page has a font too small 😅 (Remember guys, is a clone, so make it look like a clone);
image

## Please make the required changes and resubmit for a new code review once you are ready💪

## As always, I wish you all the very best and see you both at the top🚀


👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

  
<sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

**Comment by Kingobaino1** [comment]
We have used vw for the font which are below these two pictures but now we corrected this. But now it has a different structure comparing to the previous one and it is responsive now.

**Review by MuriloRoque — APPROVED** [review]
# Status: Approved :green_circle: 

## Hello @Kingobaino1 and @hamayun-cpu  :wave: 

## Thank you for making all of the required changes, your project is approved :clap: :clap: :clap: 

## You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**